packages/fxa-payments-server/server/config/index.js

```diff
@@ -117,9 +117,9 @@ const conf = convict({
       env: 'GA_SUPPORTED_STRIPE_PRODUCT_IDS',
       format: String,
     },
-    testMode: {
+    debugMode: {
```

Comment:
Renaming the config var, but keeping the `env` name the same, should ensure that we don't need to update any SRE environment variables.